src/__experimental__/components/date/date-picker.component.js

```diff
@@ -70,9 +70,8 @@ const DatePicker = (props) => {
   if (!didMount) {
     return null;
   }
-
   return (
-    <Portal onReposition={ () => didMount && setContainerPosition(getContainerPosition(window, props.inputElement)) }>
+    <Portal>
```

Comment:
I think removing that handler will introduce a bug in which after screen resizing (e.g. orientation change) the opened DatePicker won't reposition.

Reply:
You are correct my good man 😞, will revert it